utils/defaults/defaults.go

```diff
@@ -42,6 +42,9 @@ const (
 	DefaultBurst int = 80
 	// DefaultAwsLoadBalancerPageSize is the default page size used when calling aws to get load balancers by DNS name
 	DefaultAwsLoadBalancerPageSize = int32(300)
+	// DefaultMetricCleanupDelay is the default time to delay metrics removal upon object removal, gives time for metrics
+	// to be collected
+	DefaultMetricCleanupDelay = int32(65)
```

Comment:
how is this value calculated?

Reply:
We decided to start with a hard coded value at 65 seconds because prometheus default scrap setting is 60 seconds. This then allows prometheus to scrap the last metrics before we remove them. Eventually this might get asked to be configurable until then default of 65 seconds makes sense.